fixture_setup/src/load.rs

```diff
@@ -0,0 +1,123 @@
+use forte_core::context;
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+
+use dotenv;
+use toml;
+
+use diesel::associations::HasTable;
+use diesel::prelude::*;
+
+use errors::*;
+use forte_core::models::*;
+use source_models::*;
+
+pub fn load() -> Result<()> {
+    dotenv::dotenv().ok();
+
+    let path = Path::new("./node_modules/@forte-music/schema/fixtures");
+    if !path.is_dir() {
+        return Err(
+            "The fixtures can't be found. This command must be run from the fixture_setup \
+             directory of the source after running `yarn install`."
+                .into(),
+        );
+    }
+
+    let pool = context::init_pool().map_err(|err| {
+        let description = err.description();
+        let converted: Error = description.into();
+
+        converted
+    })?;
+    let db = pool.get()?;
+
+    load_from_folder(&path, &db)?;
+
+    Ok(())
+}
+
+fn load_from_folder(path: &Path, conn: &SqliteConnection) -> Result<()> {
+    let files = path.read_dir()?;
+    for file in files {
+        let file = file?;
+        let path = file.path();
```

Comment:
You don't need to shadow `file` with a temporary variable.

Reply:
How else would you do it?